.github/workflows/run-tests.yml

```diff
@@ -169,6 +169,20 @@ jobs:
           sudo chmod a+x neofs-s3-gw
         working-directory: neofs-testcases
 
+      - name: Checkout xk6-neofs repository
```

Comment:
at first I thought that because of the absence of arm binaries, but this workflow is only for amd64, so yes, no good reason, I will replace with the binary

Reply:
We'll provide ARM binaries for the next release.

Reply:
I updated it and reverted. We actually need the repo itself as well. For scenarios directory - https://github.com/nspcc-dev/xk6-neofs/tree/master/scenarios. Otherwise we need to reimplement these scenarios in python, which we can do, but I'm not sure if we need it.